src/library/scala/collection/mutable/HashTable.scala

```diff
@@ -131,7 +131,7 @@ trait HashTable[A, Entry >: Null <: HashEntry[A, Entry]] extends HashTable.HashU
   protected def findEntry(key: A): Entry =
     findEntry0(key, index(elemHashCode(key)))
 
-  private[this] def findEntry0(key: A, h: Int): Entry = {
+  protected[this] final def findEntry0(key: A, h: Int): Entry = {
```

Comment:
These two parent methods needed to be accessed from `HashMap` to avoid calculation the hash code twice.
`final` was added to keep the semantics, changing only the visibility of the method.